workflows/VGP-assembly-v2/Pre-Curation_PretextMap/PretextMap_Generation-tests.yml

```diff
@@ -0,0 +1,84 @@
+- doc: Test outline for PretextMap-Generation-from-2-haplotypes
+  job:
+    Haplotype 1:
+      class: File
+      location: https://zenodo.org/records/14218240/files/Haplotype%201.fasta
+      filetype: fasta
+    Haplotype 2:
+      class: File
+      location: https://zenodo.org/records/14218240/files/Haplotype%202.fasta
+      filetype: fasta
+    HiC reads - forward:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: HiC forward reads
+        location: https://zenodo.org/records/14218240/files/Hi-C%20forward%20reads.fastqsanger.gz
+    HiC reads - reverse:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: HI-C reverse reads
+        location: https://zenodo.org/records/14218240/files/HI-C%20reverse%20reads.fastqsanger.gz
+    PacBio reads collection:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: PacBio reads.fastq.gz
+        location: https://zenodo.org/records/14218240/files/PacBio%20reads%20collection.fastq.gz
+    Do you want to add suffixes to the scaffold names?: true
+    Haplotype 1 suffix: hap1
+    Haplotype 2 suffix: hap2
+    Do you want to trim the Hi-C data?: true
+    Telomere repeat to suit species: CCCTAA
+  outputs:
+    Merged haplotypes:
+      asserts:
+        has_text:
+          text: ">scaffold_10.hap1"
+        has_text:
+          text: ">scaffold_10.hap2"
+    Gaps Bed:
+      asserts:
+        has_text:
+          text: "scaffold_1.hap1	4100376	4100576"
+        has_text:
+          text: "scaffold_10.hap2	137138839	137139039"
+    seqtk-telo:
+        has_text:
+          text: "scaffold_10.hap2	0	11012	139653677"
+    Gaps Bedgraph:
+        has_text:
+          text: "scaffold_10.hap2	137138839	137139039	200"
+    BigWig coverage:
+      asserts:
+        has_size:
+          value : 820000
+          delta: 5000
+    Telomeres Bedgraph:
+        has_text:
+          text: "scaffold_10.hap2	0	11012	11012"
+    Trimmed Hi-C forward reads:
+      asserts:
+        has_size:
+          value : 143000000
+          delta: 1000000
+    Trimmed Hi-C reverse reads:
+      asserts:
+        has_size:
+          value : 148000000
+          delta: 1000000
+    Merged HiC alignments:
+      asserts:
+        has_size:
+          value : 71000000
+          delta: 3000000
+    graph_cov_telo_gaps:
+      asserts:
+        has_size:
+          value : 3000000
+          delta: 1000000
+
```

Comment:
Can you standardize the labels in terms of uppercase and lowercase, remove the underscores from graph_cov_telo_gaps and spell out abbreviations ? Don't forget to make the changes here and in the workflow